src/PanelWindow.vala

```diff
@@ -81,8 +81,6 @@ public class Wingpanel.PanelWindow : Gtk.Window {
 
         key_controller = new Gtk.EventControllerKey (this);
         key_controller.key_pressed.connect (on_key_pressed);
-
-        panel.size_allocate.connect (update_panel_dimensions);
```

Comment:
Maybe instead for now we do this?
```
if (!Utils.is_wayland ()) {
    panel.size_allocate.connect (update_panel_dimensions);
}
```

Reply:
Sure that works, sorry kinda forgot about this PR :grimacing: